dialogue-apache-hc4-client/src/main/java/com/palantir/dialogue/hc4/ApacheHttpClientChannels.java

```diff
@@ -385,7 +406,9 @@ public CloseableClient build() {
                             .build())
                     .setDefaultSocketConfig(socketConfig)
                     .evictIdleConnections(idleConnectionTimeoutMillis, TimeUnit.MILLISECONDS)
```

Comment:
how does this interact with the idleConnectionEvictor we provide?

Reply:
it's actually ignored at the moment because we create the idleConnectionEvictor ourselves. I left it for specificity, but we can go either way.

Reply:
can we remove it? Its a bit confusing to set config that we intend to ignore

Reply:
Removed a few other duplicates as well.